api_tests/institutions/views/test_institution_department_list.py

```diff
@@ -136,6 +136,26 @@ def test_get(self, app, url, admin, institution, populate_counts):
             'links': {'self': f'http://localhost:8000/v2/institutions/{institution._id}/metrics/departments/'}
         }]
 
+        # Tests CSV Export
+        headers = {
+            'accept': 'text/csv'
+        }
+        resp = app.get(url, auth=admin.auth, headers=headers)
+
+        assert resp.status_code == 200
+        assert resp.headers['Content-Type'] == 'text/csv; charset=utf-8'
+        response_body = resp.text
+        rows = response_body.split('\r\n')
+        header_row = rows[0].split(',')
+        new_department_row = rows[1].split(',')
+        smaller_department_row = rows[2].split(',')
+        na_row = rows[3].split(',')
```

Comment:
thinking of ways things could break -- should we test e.g. values that contain commas?

Reply:
Values containing commas is a good thing to think about. Would we allow user names, department names, or institution names with commas? Unless we add more functionality to modify the delimiter used for the CSV, it seems like that may be an acknowledged limitation.

Reply:
We would not be able to stop people from using commas, but python's CSV reader/writer should handle those cases. It would probably be a good idea to use that for the test instead of splitting on comma, and include a department or a name with a comma.